libcloud/loadbalancer/drivers/rackspace.py

```diff
@@ -349,20 +349,42 @@ def ex_list_protocols_with_default_ports(self):
         return self._to_protocols_with_default_ports(
             self.connection.request('/loadbalancers/protocols').object)
 
-    def list_balancers(self, ex_member_address=None):
+    def list_balancers(self, ex_member_address=None, ex_status=None,
+                       ex_changes_since=None, ex_params={}):
         """
         @inherits: :class:`Driver.list_balancers`
 
         :param ex_member_address: Optional IP address of the attachment member.
                                   If provided, only the load balancers which
                                   have this member attached will be returned.
         :type ex_member_address: ``str``
+
+        :param ex_status: Optional. Filter balancers by status
+        :type ex_status: ``str``
+
+        :param ex_changes_since: Optional. List all load balancers that have
+                                 changed since the specified date/time
+        :type ex_changes_since: ``str``
+
+        :param ex_params: Optional. Set parameters to be submitted to the API
+                          in the query string
+        :type ex_params: ``dict``
         """
+
         params = {}
 
         if ex_member_address:
             params['nodeaddress'] = ex_member_address
 
+        if ex_status:
+            params['status'] = ex_status
+
+        if ex_changes_since:
+            params['changes-since'] = ex_changes_since
+
+        for key, value in ex_params.iteritems():
```

Comment:
`dict.iteritems()` is not valid in Python 3, you must use items().

Which probably highlights that this code doesn't have a supporting unit test as TravisCI should have picked that up. please can you add a test.